.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+    "presets": [
+      "@babel/preset-react",
+      ["@babel/preset-env", {
+        "modules": false,
+        "targets": {
+          "browsers": ["last 3 versions"]
```

Comment:
What do we want to do for this? Relevant discussion: https://github.com/eslint/eslint.github.io/issues/543

Reply:
If we don't wan to think about this yet, we can also specify `forceAllTransforms: true`, and it will compile everything to be ES5-compatible.